src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/CellMoveIntegrationTest.kt

```diff
@@ -97,15 +90,22 @@ class CellMoveIntegrationTest : IntegrationTest() {
 
   @Test
   fun `should return cell move details`() {
-    whenever(cellMoveReasonRepository.findById(any())).thenReturn(
-      Optional.of(
-        CellMoveReason(
-          bookingId = SOME_BOOKING_ID,
-          bedAssignmentsSequence = SOME_BED_ASSIGNMENT_SEQUENCE,
-          caseNoteId = SOME_CASE_NOTE_ID
+    stubCellMove()
+
+    webTestClient.post()
+      .uri("/cell/make-cell-move")
+      .bodyValue(
+        mapOf(
+          "bookingId" to SOME_BOOKING_ID,
```

Comment:
the 'SOME_' adds clutter in my view

Reply:
Agreed. I've done it a few places so I'll clean them up in future PRs.